mob.go

```diff
@@ -645,6 +645,12 @@ func getCachedChanges() string {
 	return strings.TrimSpace(silentgit("diff", "--cached", "--stat"))
 }
 
+func makeWipCommit() {
+	commitMsg := configuration.WipCommitMessage
+	git("add", "--all")
+	git("commit", "--allow-empty", "--message", commitMsg, "--no-verify")
```

Comment:
Should not allow empty commits at the moment. Only necessary for s special case when having the with argument. Can you remove that, please?

Reply:
Yep, missed that -- done!